src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -14,16 +16,25 @@ import {
   TouchableOpacity,
   View
 } from 'react-native';
+import {useRoute} from '@react-navigation/core';
 
 import AnonymousChatInfoHeader from '../../components/Header/AnonymousChatInfoHeader';
 import AnonymousIcon from '../ChannelListScreen/elements/components/AnonymousIcon';
-import useAnonymousChatInfoScreenHook from '../../hooks/screen/useAnonymousChatInfoHook';
-import useProfileHook from '../../hooks/core/profile/useProfileHook';
+import BlockComponent from '../../components/BlockComponent';
+import ChannelImage from '../../components/ChatList/elements/ChannelImage';
+import ModalAction from '../GroupInfo/elements/ModalAction';
+import ModalActionAnonymous from '../GroupInfo/elements/ModalActionAnonymous';
+import dimen from '../../utils/dimen';
+import useChatInfoScreenHook from '../../hooks/screen/useChatInfoHook';
+import useUserAuthHook from '../../hooks/core/auth/useUserAuthHook';
+import {CHANNEL_GROUP, GROUP_INFO, SIGNED} from '../../hooks/core/constant';
+import {Context} from '../../context';
 import {Loading} from '../../components';
 import {ProfileContact} from '../../components/Items';
 import {colors} from '../../utils/colors';
 import {fonts, normalize, normalizeFontSize} from '../../utils/fonts';
-import {trimString} from '../../utils/string/TrimString';
+import {getChatName} from '../../utils/string/StringUtils';
+import {isContainUrl} from '../../utils/Utils';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been modified and new ones are added. However, it's not clear why the `useProfileHook` is replaced with `useUserAuthHook`. If `useUserAuthHook` provides more functionality or better encapsulates the user authentication logic, then this change makes sense. But if both hooks provide similar functionality, then this change might be unnecessary.



<!-- This is an auto-generated comment by OSS CodeRabbit -->